region/norfair/east/Lava Dive Room.json

```diff
@@ -442,6 +442,27 @@
         "Then Walljump again to exit the Lava and reach the left ledge."
       ]
     },
+    {
+      "link": [2, 3],
```

Comment:
The part that is off is ending at [3] instead of [1].  
At 3 you get some heat frames for running to and opening the door and works better for missile doors.
But the artificial morph could open a PB door.

Reply:
I didnt realize this would be possible without Varia. My earlier attempts looked like it would require it, but thats cool. I'll throw 50 heat frames to escape the room and put a note on it to come back to it. Havent completely decided how to deal with heated g-mode and exiting the room yet.